### PR TITLE
Add images

### PR DESCRIPTION
**Pull request by chhoumann**

### CODE REVIEW

File: report/sections/Appendix/frontend-images.tex [hunk 1]
```diff
@@ -0,0 +1,35 @@
+\chapter{Platform demonstration} \label{chap:images}
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale=0.1]{home.png}
```
Comment (MrHonket) [inline report/sections/Appendix/frontend-images.tex:4]:
```suggestion
   \frame{\includegraphics[scale=0.1]{home.png}}
```

File: report/sections/Appendix/frontend-images.tex [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    \caption{Home page as a logged-in user.}
+    \label{fig:Home page}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale=0.1]{syllabi.png}
```
Comment (MrHonket) [inline report/sections/Appendix/frontend-images.tex:11]:
```suggestion
   \frame{\includegraphics[scale=0.1]{syllabi.png}}
```

File: report/sections/Appendix/frontend-images.tex [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    \caption{Syllabi overview as a logged-in student.}
+    \label{fig:syllabi}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale=0.1]{problemsets.png}
```
Comment (MrHonket) [inline report/sections/Appendix/frontend-images.tex:18]:
```suggestion
    \frame{\includegraphics[scale=0.1]{problemsets.png}}
```

File: report/sections/Appendix/frontend-images.tex [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale=0.1]{exercises.png}
+    \caption{Exercise overview.}
+    \label{fig:exercise_overview}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[scale=0.1]{teacher_view.png}
```
Comment (MrHonket) [inline report/sections/Appendix/frontend-images.tex:32]:
```suggestion
    \frame{\includegraphics[scale=0.1]{teacher_view.png}}
```

**Review by MrHonket — CHANGES_REQUESTED** [review]
The pictures looks good overall, requested a few additional settings to improve the look

File: report/sections/Chapter 06 - Test runner/Chapter 6.tex [hunk 5]
```diff
@@ -20,12 +27,28 @@ \section{Client-Server Interaction}
 However, we want to eventually display the test code to the user so they may gain a better understanding of the solution requirements, although this will not be done in this iteration of the project.
 
 To handle this mutation, the Next.js server requests the Test Runner to process the given solution and test code.
-This process is described in section \label{sec:test_runner_process} below.
+This process is described in section \ref{sec:test_runner_process} below.
 The response of this request to the Test Runner is validated and stored as a submission in the database.
 This allows users and teachers to access previous submissions, which includes code and a value indicating whether the submission satisfied the exercise tests.
```
Comment (Ivikhostrup) [inline report/sections/Chapter 06 - Test runner/Chapter 6.tex:32]:
```suggestion
This allows users and lecturers to access previous submissions, which includes code and a value indicating whether the submission satisfied the exercise tests.
```

**Review by Ivikhostrup — APPROVED** [review]

**Review by Ivikhostrup — APPROVED** [review]

**Pull request merged**